composer.json

```
@@ -24,7 +24,8 @@
     ],
     "require": {
         "php": ">=7.2.0",
-        "ext-iconv": "*"
+        "ext-iconv": "*",
+        "psr/log": "*"
```

For dependencies that are actual packages (instead of PHP extensions), please always use specific versions:

```suggestion
        "psr/log": "^1.0.0 || ^2.0.0 || ^3.0.0"
```